test/migration_repos_workflow_spec.rb

```diff
@@ -39,7 +39,7 @@
 
       let(:set_success_expectations) do
```

Comment:
It's not related with the PR, but it does not seem to be a correct use case for `let`. Writing a helper method will be a better solution. But, as I said, it's out of the scope of this PR.